cfy_manager/components/manager/manager.py

```diff
@@ -43,7 +43,7 @@ def _install(self):
         self._create_sudoers_file_and_disable_sudo_requiretty()
         if self.service_type == 'supervisord':
             self._allow_run_supervisorctl_command()
-        self._set_selinux_permissive()
+        self._set_selinux_enforcing()
```

Comment:
I'm not convinced we need to do this- I think we should set up the policies but leave it up to the user whether they have SELinux enforcing on their images or not. If they don't then that's fine, it won't impact our general functionality and we won't mess about with their preferred settings (which they might have set due to some other software they run that they haven't set up selinux for- e.g. monitoring software or something).

Reply:
agreed